mlflow/utils/validation.py

```diff
@@ -13,7 +13,7 @@
 _VALID_PARAM_AND_METRIC_NAMES = re.compile(r"^[/\w.\- ]*$")
 
 # Regex for valid run IDs: must be a 32-character hex string.
```

Comment:
Micronit: would be nice to update this comment to reflect the updated validation, e.g.

"Regex for valid run IDs: must be an alphanumeric string of length 0 to 255" 

BTW, was it intentional to allow zero-length run IDs? Empty-string run IDs will break with the FileStore and IMO could be somewhat confusing for users to deal with - sorry if I missed discussion about this point elsewhere, please let me know if that's the case

Reply:
zero-length run IDs are not supported, the first [] forces the first character to be alphanumeric, and is not optional. So minimum length is 1

Thanks for catching the stale comment!